packages/sources/tiingo/test-payload.json

```diff
@@ -10,7 +10,7 @@
       "endpoint": "realized-vol"
     },
     {
-      "aprTerm": "30day",
+      "aprTerm": "30Day",
```

Comment:
Out of curiosity, why did this change?

Reply:
Tried these requests on our prod Tiingo EA and this one was failing.
Unfortunately this wasn't the cause of the soak test failures.

Reply:
here's the error with the original `"aprTerm": "30day"`:
```json
{
    "status": "errored",
    "statusCode": 400,
    "error": {
        "name": "AdapterError",
        "message": "[Param: aprTerm] input is not one of valid options (30Day,90Day)"
    }
}
```